qiskit/primitives/__init__.py

```diff
@@ -464,12 +466,16 @@
     DataBin,
     PrimitiveResult,
     PubResult,
+    SamplerPubResult,
     EstimatorPubLike,
     SamplerPubLike,
     BindingsArrayLike,
     ObservableLike,
     ObservablesArrayLike,
+    Shaped,
```

Comment:
@ihincks is this intended to be a public interface. I assume so since it's documented in a release note. But it'd be good to get an ack here.

Reply:
ack 👍